### DISCUSSION TITLE
Is there a way to highlight a custom element similar to emphasis in vscode?

### DISCUSSION THREAD

**Discussion by LokiMidgard** [remarkjs/remark#1373] [Q&A]
When creating a plugin, that adds a custom element, similar to [remark-ins](https://github.com/ipikuka/remark-ins).
Is there a way to implement it in a way the vscode extension will highlight it in some way. like the emphasis or strong?

**Comment by wooorm** [comment]
That’s one of the main reasons not to constantly add random things to markdown, but instead benefit from either directives or MDX: other tools don’t understand things.

As for how to integrate with syntax highlighting, I think that has to do with VS Code itself. I believe TextMate grammars can indeed extend each other, but not exactly sure how.

**Comment by remcohaszing** [comment]
I’m not sure how it should work yet, but I’m open to discussing this feature for https://github.com/remcohaszing/vscode-markdown-decorations.

**Reply by LokiMidgard** [reply]
Ok I'll open an issue there to to start a discussion :)

**Reply by LokiMidgard** [reply]
https://github.com/remcohaszing/vscode-markdown-decorations/issues/1
